src/ovh-angular-mondial-relay/ovh-angular-mondial-relay.view.html

```diff
@@ -1,113 +1,118 @@
+                <ul class="list-group">
+                    <li class="list-group-item"
+                        data-ng-repeat="relay in $ctrl.foundRelays track by relay.id"
+                        data-ng-mouseover="$ctrl.markerHover($index)"
+                        data-ng-click="$ctrl.select(relay)"
+                        data-ng-class="{ 'active': relay.selected, 'marker-hover': relay.markerHover }">
+                        <div class="relay-cell">
+                            <div class="oui-radio">
+                                <input class="oui-radio__input"
+                                       name="oui-radio"
+                                       value="oui-radio-{{$index}}"
+                                       id="oui-radio-{{$index}}"
+                                       type="radio">
+                                <label class="oui-radio__label-container"
+                                       for="oui-radio-{{$index}}">
+                                    <span class="oui-radio__label">
+                                        <span class="mondial-marker"
+                                              data-ng-bind="relay.marker">
+                                        </span>
+                                        <strong class="mondial-relay-name"
+                                                data-ng-bind="relay.name">
+                                        </strong>
+                                        <span class="mondial-relay-caret glyphicon"
+                                              data-ng-class="'glyphicon-menu-' + (relay.selected? 'up': 'down')"></span>
+                                    </span>
+                                    <span class="radio-description">
+                                        <span class="radio-description-relay">
+                                            <span class="mondial-relay-address"
+                                                  data-ng-bind="relay.address">
+                                            </span>
+                                             –
+                                            <span class="mondial-relay-city">
+                                                <span data-ng-bind="relay.zipcode"></span>
+                                                <span data-ng-bind="relay.city"></span>
+                                            </span>
+                                        </span>
+                                        <span data-ng-if="relay.selected">
+                                            <ul class="mondial-opening">
+                                                <li class="row"
+                                                    data-ng-repeat="item in relay.opening track by $index"
+                                                    data-ng-if="item.days">
+                                                    <span class="col-xs-4 text-right"
```

Comment:
Can't you use columns instead of `text-right` ?

Reply:
Both are used (: